spec/mailers/previews/anonymous_mailer_preview.rb

```diff
@@ -0,0 +1,15 @@
+class AnonymousMailerPreview < ActionMailer::Preview
```

Comment:
should we make an `anonymous_mailer_preview_spec.rb` that ensures that all mail methods have corresponding preview methods?

Reply:
Yeah, that sounds like a good idea. I've found that we've built quite a lot around UserMailer as the assumed "only" mailer class, so introducing a new one has proven to be quite a challenge 😬

Reply:
Added preview spec in 1874b39e4a